src/redux/configureStore.js

```diff
@@ -0,0 +1,30 @@
+import {
+  createStore, compose, combineReducers, applyMiddleware,
+} from 'redux';
+import logger from 'redux-logger';
+import {
+  fetchDataMiddleware,
+  detailDataReducer,
+  filteredDataReducer,
+  loadingStateReducer,
+  loadingChangeStateMiddleware,
+} from './home/home';
+
+const reducer = combineReducers({
+  detailDataReducer,
+  filteredDataReducer,
+  loadingStateReducer,
+});
+const composedEnhancer = compose(
+  applyMiddleware(fetchDataMiddleware),
+  applyMiddleware(loadingChangeStateMiddleware),
+  applyMiddleware(logger),
+);
```

Comment:
**[Optional]**
- [ ] Please Do not log redux data on the browser console as it may cause site security issues. Kindly clear the logged data and the errors displayed on the browser console.